lib/simplecov/command_guesser.rb

```diff
@@ -48,8 +48,8 @@ def from_defined_constants
           "RSpec"
         elsif defined?(Test::Unit)
           "Unit Tests"
-        elsif defined?(MiniTest)
-          "MiniTest"
+        elsif defined?(Minitest)
```

Comment:
why not both?

Reply:
I'd be OK with both, but since the original constant was going to be completely removed, and the new one has been current for _ages_, it just seemed like debt.

But if we want it back I don't see the harm :)

Reply:
@adam12 yes please add it back :) It's a small price to pay to maybe still support some people with very old versions.

Reply:
I'll do just that. Thanks!